libs/common-utils/jest.config.ts

```diff
@@ -6,6 +6,7 @@ export default {
     '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nx/react/plugins/jest',
     '^.+\\.[tj]sx?$': ['babel-jest', { presets: ['@nx/react/babel'] }],
   },
+  // setupFilesAfterEnv: ['../../jest.setup.ts'],
```

Comment:
This property is enabled in other libs. Should we keep it here as well?

Reply:
thx https://github.com/cowprotocol/cowswap/pull/3301/commits/1f9c9f969bbce6875cf4af960bc3d4a897e374e5